defaults/main.yml

```diff
@@ -10,6 +10,7 @@ rundeck_database_user: 'rundeck'
 rundeck_database_pass: 'rundeck'
 rundeck_users: []  # list of users as dictionary of name, password and list of roles
 rundeck_plugins: []  # list of plugin urls that are downloaded and installed into the rundeck libext.
+rundeck_extra_bootstrap: []  # list of jar urls that are downloaded and installed into the rundeck bootstrap.
```

Comment:
I don't think this is really needed for this addition and the installation of the jars should be a part of the crowd setup.